compose/compose-file/index.md

```diff
@@ -691,6 +699,23 @@ If you have further questions, please refer to the discussion on the GitHub
 issue [docker/compose/4513](https://github.com/docker/compose/issues/4513){: target="_blank" class="_"}.
 {: .important}
 
+##### Generic Resources
+
+Generic Resources are a way to select the kind of nodes your task can land on.
+
+In a swarm cluster, nodes can advertise Generic resources as Discrete values or
```

Comment:
We need to decide if they are Generic Resources or Generic resources or generic resources. 👼

Reply:
Agreed, I went with `Generic resources` except for the title.